packages/kbn-handlebars/scripts/check_for_upstream_updates.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+TMP=.tmp-handlebars
+HASH_FILE=packages/kbn-handlebars/src/spec/.upstream_git_hash
+
+# Try to detect Windows environment (I've not tested this!)
+if [[ "$OSTYPE" == "msys" ]]; then
+  # Windows environment
+  DEVNULL=NUL
+else
+  # Everything else (including Cygwin on Windows)
+  DEVNULL=/dev/null
+fi
```

Comment:
__question:__ why do we need this? It looks like we don't use `DEVNULL` anywhere, and it's just a leftover from the previous script

Reply:
Ups, forgot to remove. Nice catch 👍